client/index.html

```diff
@@ -20,9 +20,9 @@
       label,
       text,
       div {
-        font-family: "Roboto Condensed", "Helvetica Neue", "Helvetica", "Arial",
-          sans-serif;
+        font-family: "Inter", "Helvetica Neue", "Helvetica", "Arial", sans-serif;
         font-size: 14px;
+        letter-spacing: -0.006em;
```

Comment:
why this change?

Reply:
Requested by Harley with the font change!